tsconfig.json

```diff
@@ -22,6 +22,6 @@
     /* Skip type checking of declaration files. */
     "skipLibCheck": true,
     /* Disallow inconsistently-cased references to the same file. */
-    "forceConsistentCasingInFileNames": true
-  }
+    "forceConsistentCasingInFileNames": true,
+  },
```

Comment:
```suggestion
    "forceConsistentCasingInFileNames": true
  }
```

Reply:
This was changed by ./tools/code_style.sh